libgraph/src/BuildGraph.cpp

```diff
@@ -1895,7 +1896,11 @@ katana::WritePropertyGraph(
         "edge prop: ({}) {}", field->type()->ToString(), field->name());
   }
 
-  auto result = prop_graph.Write(dir, "libkatana_graph");
+  // Since this function is only used by tools under
+  // external/katana/tools/graph-convert/, but not any other graph
+  // operations that can potentially be involved in transactions,
+  // we commit the RDGManifest here.
+  auto result = prop_graph.Write(dir, "libkatana_graph", txn_ctx);
   if (!result) {
     return result.error().WithContext("writing to fs");
   }
```

Comment:
I would write this as `auto result = KATANA_CHECKED(prop_graph.Write(dir, "libkatana_graph", txn_ctx));`
My rule of thumb is to only output a context message if I have data to print in the message.